app/src/main/java/com/nilhcem/blenamebadge/ui/badge_preview/PreviewBadge.kt

```diff
@@ -95,30 +108,216 @@ class PreviewBadge : View {
         // Draw Cells
         for (i in 0 until badgeHeight) {
             for (j in 0 until badgeWidth) {
-                if (i < checkList?.size ?: -1 && j < checkList?.get(i)?.list?.size ?: -1 && checkList?.get(i)?.list?.get(j) == true) {
-                    ledEnabled?.bounds = cells[i].list[j]
-                    ledEnabled?.draw(canvas)
+                if (ifFlash == true) {
+                    val aI: Int = animationIndex % 800
+                    val valid: Boolean = aI > 400
+
+                    if (valid && i < checkList?.size ?: -1 && j < checkList?.get(i)?.list?.size ?: -1 && checkList?.get(i)?.list?.get(j) == true) {
```

Comment:
Remove the `== true` from here too.

Reply:
Same, Sir. Nullable Property, won't work without checking against the value

Reply:
If that's the case, then ok. Resolve similar comments then.